### PR TITLE
PLP-38:getAllCompanies

### PR DESCRIPTION
**Pull request by PSY27**
query to getAllCompanies details in the dashboard

### CODE REVIEW

File: server/src/graphql/resolvers/company/queries.graphql.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { ApolloContext } from "../../../context";
+import { Company,QueryResolvers } from "../../../types/graphql";
+
+export const queries: QueryResolvers<ApolloContext, Company> = {
+	getAllCompanies: async (_, {}, { prisma }) => {
```
Comment (Nishan212) [inline server/src/graphql/resolvers/company/queries.graphql.ts:5]: { prisma }: ApolloContext

File: server/src/graphql/resolvers/company/queries.graphql.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+		const companies: Company[] | null = await prisma.company.findMany({
+
```
Comment (Nishan212) [inline server/src/graphql/resolvers/company/queries.graphql.ts:7]: PLease add prettier extension

**Review by Nishan212 — CHANGES_REQUESTED** [review]

File: server/src/graphql/resolvers/company/queries.graphql.ts (hunk 1 above)
Comment (PSY27) [reply server/src/graphql/resolvers/company/queries.graphql.ts]: done

File: server/src/graphql/resolvers/company/queries.graphql.ts (hunk 2 above)
Comment (PSY27) [reply server/src/graphql/resolvers/company/queries.graphql.ts:7]: done

**Review by Nishan212 — APPROVED** [review]

**Pull request merged**